src/collections/initiatives/recQcaxqWsxtnljqe.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Inclusive AI for HR
+id: recQcaxqWsxtnljqe
+permalink: /initiatives/recQcaxqWsxtnljqe/
+eventDate: 2020-12-01T04:00:00.000Z
+coverImageUrl: /images/uploads/FWDlogoEforprocessing4x.png
```

Comment:
There are a few typos in image names in these initiatives markdown files, which causes these images are not shown in the content editing fields. For example, the filename for this image in the local directory is `FWDlogoBforprocessing4x1.png`. Same type of typo for the preview image in the "Coding to Learn and Create" workshop.

Could you go through the initiatives collection and make sure all images work as expected? Thanks.